.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.cy.js"],
```

Comment:
Adding `cypress.config.js` in the files section here can help with errors.

Reply:
Thank you :) I will implement this change.